PSReadLine/PlatformWindows.cs

```diff
@@ -201,6 +202,14 @@ internal static void Init(ref ICharMap charMap)
                 EnableAnsiInput(ref charMap);
             }
 
+            // Is the TerminateOrphanedConsoleApps feature enabled?
+            if (_allowedPids != null)
+            {
+                // We are about to disable Ctrl+C signals... so if there are still any
+                // console-attached children, the shell will be broken until they are
+                // gone, so we'll get rid of them:
+                TerminateStragglers();
+            }
```

Comment:
Minor style suggestion -- add an empty line after the closing bracket.
```suggestion
            }

```